src/sass/table/table.scss

```diff
@@ -1,10 +1,40 @@
 @import "../variables/variables";
 
 :root {
```

Comment:
Seems like there are a lot of variables here. 
I know we have an issue where having too many variables can cause performance problems because they cannot be mangled and add to the size of the bundle.
Do we need all these variables? Could we simplify them somehow?
Maybe we can have these as sass variables and then change the "height" in the code to be a token mixin and have the first variable be the override for that (especially when we get to theming)